include/llvm/Config/config.h

```diff
@@ -104,7 +111,10 @@
 #define HAVE_LIBPTHREAD 1
 
 /* Define to 1 if you have the `pthread_getname_np' function. */
+#if defined(__ANDROID__)
+#else
 #define HAVE_PTHREAD_GETNAME_NP 1
+#endif
```

Comment:
Added in Android 8 as long as _GNU_SOURCE is defined, but for backwards compatibility commented out.